src/ui-kit/wrappers/label-wrapper/label-wrapper.template.html

```diff
@@ -11,7 +11,7 @@
       <ng-content select="[label-right]"></ng-content>
   </label>
   
-  <div *ngIf="hint" class="toggle-more">
+  <div *ngIf="hint" class="toggle-more" aria-live="polite">
```

Comment:
Adding aria-live ensures JAWS will announce the hint anytime more/less is toggled.